packages/operator/pkg/config/deployment.go

```diff
@@ -64,8 +64,8 @@ type ModelDeploymentConfig struct {
 	// Default connection ID which will be used if a user doesn't specify it in a model deployment
 	DefaultDockerPullConnName string     `json:"defaultDockerPullConnName"`
 	Edge                      EdgeConfig `json:"edge"`
-	// Kubernetes node selector for model deployments
-	NodeSelector map[string]string `json:"nodeSelector"`
+	// Node pools to run training tasks on
```

Comment:
Node pools to run deployment tasks on
(check for packaging plz)

Reply:
done